operator/olm-base/metadata/annotations.yaml

```diff
@@ -0,0 +1,9 @@
+annotations:
+  # Core bundle annotations.
+  operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+  operators.operatorframework.io.bundle.manifests.v1: manifests/
+  operators.operatorframework.io.bundle.metadata.v1: metadata/
+  operators.operatorframework.io.bundle.package.v1: keycloak-operator
+  operators.operatorframework.io.bundle.channels.v1: alpha
+  operators.operatorframework.io.bundle.channel.default.v1: alpha
```

Comment:
jFYI – no need to change it in this PR. The channel name will have to be different. Probably `candidate` for releases and `nightly` for nightlies. I.e. will have to be variable. We can do it as a follow-up.